test/views/ConfirmationAnswerPageViewSpec.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import views.behaviours.ViewBehaviours
+import views.html.ConfirmationAnswerPageView
+
+class ConfirmationAnswerPageViewSpec extends ViewBehaviours {
+  val index = 0
+
+  "ConfirmationAnswerPage view" must {
+
+    val view = viewFor[ConfirmationAnswerPageView](Some(emptyUserAnswers))
+
+    val applyView = view.apply(Nil, "", "")(fakeRequest, messages)
+
+    behave like normalPage(applyView, "confirmationAnswerPage")
```

Comment:
Do we want to add any view specs as we're rendering a lot of information? Worth having a conversation with @JohnWhitfield to understand how much of this is done between unit tests and journey tests.

Reply:
Done